bundles/org.openhab.binding.growatt/src/main/java/org/openhab/binding/growatt/internal/cloud/ApiException.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.growatt.internal.cloud;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link ApiException} is thrown if a call to the Growatt cloud API server fails.
+ *
+ * @author Andrew Fiddian-Green - Initial contribution.
+ */
+@NonNullByDefault
+public class ApiException extends Exception {
```

Comment:
Maybe make the name more specific: `GrowattApiException`?

Reply:
Ok